src/lightning_horovod/strategy.py

```diff
@@ -18,21 +18,39 @@
 import horovod.torch as hvd
 import torch
 import torch.nn as nn
-from lightning_fabric.plugins import CheckpointIO
-from lightning_fabric.utilities.distributed import _distributed_available
-from lightning_fabric.utilities.distributed import group as dist_group
-from lightning_fabric.utilities.types import ReduceOp
-from pytorch_lightning import Trainer
-from pytorch_lightning.accelerators import Accelerator
-from pytorch_lightning.core.optimizer import LightningOptimizer
-from pytorch_lightning.plugins.precision import PrecisionPlugin
-from pytorch_lightning.strategies.parallel import ParallelStrategy
-from pytorch_lightning.strategies.strategy import TBroadcast
-from pytorch_lightning.utilities.exceptions import MisconfigurationException
-from pytorch_lightning.utilities.rank_zero import rank_zero_only
+from lightning_utilities import module_available
 from torch import Tensor
 from torch.optim import Optimizer
 
+if module_available("lightning"):
```

Comment:
this still has the issue that you cannot use the strategy with `pytorch_lightning` if `lightning` is installed, right?

Reply:
Could be but this issue would be for any user who has installed both, right?
anyway I would land this and wait is someone cross the mentioned issue :otter: